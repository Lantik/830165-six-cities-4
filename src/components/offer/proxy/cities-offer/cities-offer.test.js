import React from 'react';
import ShallowRenderer from 'react-test-renderer/shallow';
import CitiesOffer from './cities-offer.jsx';
import Offer from '../../offer.jsx';
import {doWithoutPropTypes} from '../../../../utils/testing/testing.js';

const mock = {
  offer: {}
};

it(`<CitiesOffer/> rendered correctly`, () => doWithoutPropTypes([Offer], () => {
  const {offer} = mock;
  const renderer = new ShallowRenderer();

  renderer.render(<CitiesOffer
    offer={offer}
    onMouseEnter={() => {}}
    onOfferHeaderClick={() => {}}
  />);
  const tree = renderer.getRenderOutput();

  expect(tree).toMatchSnapshot();
}));
