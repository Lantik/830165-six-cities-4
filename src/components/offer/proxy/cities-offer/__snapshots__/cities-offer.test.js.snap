// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<CitiesOffer/> rendered correctly 1`] = `
<Offer
  articleClassName="cities__place-card"
  imageWrapperClassName="cities__image-wrapper"
  offer={Object {}}
  onMouseEnter={[Function]}
  onOfferHeaderClick={[Function]}
/>
`;
